app/controllers/school_groups/secr_controller.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module SchoolGroups
+  class SecrController < ApplicationController
+    load_and_authorize_resource :school_group
+
+    def index
+      raise CanCan::AccessDenied unless current_user.admin? || current_user.group_admin?
+
+      @start_date = MeterMonthlySummary.start_date(Time.zone.today, 2)
+      @meters = @school_group.meters.active.where('schools.active')
+      respond_to do |format|
+        format.html
+        format.csv do
+          type = params[:csv]
+          send_data csv_report(type), filename: EnergySparks::Filenames.csv("secr-#{type}")
+        end
+      end
+    end
+
+    private
+
+    def csv_headers
+      [t('common.school'),
+       'MPXN',
```

Comment:
should these all be translated?

Reply:
Yes, all of the CSV downloads from the group dashboard have translated headings, so need to handle them here too.